airbyte-integrations/connectors/source-shopify/source_shopify/scopes.py

```diff
@@ -83,16 +83,14 @@ class ShopifyScopes:
     logger = logging.getLogger("airbyte")
 
     def __init__(self, config: Mapping[str, Any]) -> None:
+        self.permitted_streams: List[str] = list(ALWAYS_PERMITTED_STREAMS)
```

Comment:
This change is odd but those were shared between all instances of the class. Adding tests highlighted that. I think in prod it wasn't an issue because this code is executed only during a command

Reply:
This is fine, to have the `instance` vars instead of the class vars.